.github/workflows/deploy-feature-azure-webapps.yml

```diff
@@ -0,0 +1,130 @@
+name: Deploy feature validation samples
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
```

Comment:
Can we remove this? The option to choose a branch should always be there by default:
![image](https://github.com/Azure/communication-ui-library/assets/2684369/fbe8a811-d1d4-47b6-a81f-f3944a649590)

Reply:
Removed